src/app.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+import Toast from "./toast.js";
+import Modal from "./modal.js";
```

Comment:
it is es6 logic. Please consider replacing it with a revealing module pattern (see lection of April, 12)